language/tools/move-mv-llvm-compiler/src/stackless/module_context.rs

```diff
@@ -848,6 +848,84 @@ impl<'mm, 'up> ModuleContext<'mm, 'up> {
         attrs
     }
 
+    // This function extracts an entry function actual arguments from
+    // AccountInfo data field.  The field is a byte array containing
+    // values of actual arguments in sequential order without gaps.
+    fn emit_entry_arguments(
+        &self,
+        fn_env: &mm::FunctionEnv,
+        accounts: &llvm::AnyValue,
+    ) -> Vec<llvm::AnyValue> {
+        if fn_env.get_parameter_count() == 0 {
+            return vec![];
+        }
+        let llcx = self.llvm_cx;
+        let vec_ty = llcx.get_anonymous_struct_type(&[
+            llcx.ptr_type(),
+            llcx.int_type(64),
+            llcx.int_type(64),
+        ]);
+        let acc_vec_ptr =
+            self.llvm_builder
+                .getelementptr(*accounts, &vec_ty.as_struct_type(), 0, "acc_vec_ptr");
+        let acc_vec_ptr =
+            self.llvm_builder
+                .load(acc_vec_ptr, llcx.ptr_type(), "acc_vec_ptr_loaded");
+        let data_ty = llcx.get_anonymous_struct_type(&[llcx.ptr_type(), llcx.int_type(64)]);
+        let acc_ty = llcx.get_anonymous_struct_type(&[
+            llcx.ptr_type(),   // key
+            llcx.int_type(64), // lamports
+            data_ty,           // data
+            llcx.ptr_type(),   // owner
+            llcx.int_type(64), // rent_epoch
+            llcx.int_type(1),  // is_signer
+            llcx.int_type(1),  // is_writable
+            llcx.int_type(1),  // executable
+        ]);
```

Comment:
It would also be good to factor his out into a function in  `RttyContext` as we have for `get_llvm_type_for_move_native_vector`.

Reply:
Done.